resources/telemetry/charts/operator/values.yaml

```diff
@@ -69,7 +69,7 @@ fluentbit:
 resources:
   limits:
     cpu: 100m
-    memory: 256Mi
+    memory: 384Mi
   requests:
     cpu: 5m
     memory: 20Mi
```

Comment:
Let's increase the requests as well. It seems unrealistic that the operator runs with only 20Mi.